### ISSUE TITLE
[Adaptive Card > InputFormwithRTL.Json]: Screen Reader is not announcing the required information while navigating to Name edit field in Arabic language.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#274]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.24.11.3.1

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS Version: 17.7.2
App Version: 1.0 (2.4.0-beta.24.11.3.1
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "InputFormWithRTL.json" control and double tap to activate it.
7. Perform right swipe and navigate to the 'Arabi' card button and double tap to expand it.
8. Navigate to the button using swipe gesture and observe the issue.

**Actual Result:**
While navigating to the name edit field in Arabic language, screen reader is not conveying the required information along with label.

**Expected Result:**
While navigating to the name edit field in Arabic language, screen reader should announce the required information along with the label.

**User Experience:**
It will impact the screen reader user as they will not able to get to know about the required field while navigating in Arabic language if the screen reader will not announce the required information.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/725bdd3b-ce18-44e8-ae20-8f6fce2c7c22

### Card JSON

```JSON
NA
```
